build.sbt

```diff
@@ -32,7 +32,7 @@ lazy val cmta = project
   .dependsOn(`cmt-core`, `cmt-core` % "test->test")
   .settings(commonSettings: _*)
   .settings(nativeImageSettings: _*)
-  .settings(Compile / mainClass := Some("com.lunatech.cmt.admin.Main"))
+  .settings(Compile / mainClass := Some("coursemgmttools.admin.Main"))
```

Comment:
I feel like `tools` is a bit redundant. I would omit it and keep`coursemgmt`.

It would then be `coursemgmt.admin` and `coursemgmt.client` which read better to me.

WDYT?

Reply:
Agreed. Makes it shorter too. `coursemgmttools` is a bit too long.

Reply:
Shortened package name to `coursemgmt`